stable/sysdig/README.md

```diff
@@ -92,6 +94,30 @@ $ helm install --name sysdig-agent-on-prem \
     stable/sysdig
 ```
 
+## Using private image registries
+
+To authenticate against an image registry you will need to store the credentials
+in a ConfigMap and add a reference to that ConfigMap.
+
+```bash
+kubectl create secret docker-registry NAME \
+ --docker-server=SERVER \
+ --docker-username=USERNAME \
+ --docker-password=TOKEN \
+ --docker-email=EMAIL
+```
+
+And you must pass its reference in a YAML value file which will be used when
+deploying the Sysdig agent Helm chart (this cannot be done using the command-line):
```

Comment:
The values YAML file will need to point to the Secret you just created (this cannot be done using the command-line):